qcore/config.py

```diff
@@ -1,22 +1,34 @@
-
 import json
 import os
 import platform
 
-hostname = platform.node()
-if hostname.startswith("ni") and len(hostname) == 8:
-    host = "maui"
-    basename = os.path.join('machine_config', 'config_maui.json')
-elif hostname.startswith("wb") and len(hostname) == 6:
-    host = "mahuika"
-    basename = os.path.join('machine_config', 'config_mahuika.json')
-else:
-    # default
-    host = "unknown"
-    basename = 'config.json'
 
+def get_host_config():
+    """
+    Determine the actual hostname eg.ni0002--> maui; maui01-->maui
+    :return: actual hostname, config josn file path
+    """
+    host_name = platform.node()
+    if (host_name.startswith("ni") and len(host_name) == 8) or host_name.startswith('maui'):  # maui
```

Comment:
comments at the end of lines are messy. pointless anyway because on the next line it says "maui"